show/main.py

```diff
@@ -383,20 +383,23 @@ def protocol():
 
 
 #
-# Inserting BGP functionality into cli's show parse-chain. The insertion point
-# and the specific BGP commands to import, will be determined by the routing-stack
-# being elected.
+# Inserting BGP functionality into cli's show parse-chain.
+# BGP commands are determined by the routing-stack being elected.
 #
 if routing_stack == "quagga":
     from .bgp_quagga_v4 import bgp
     ip.add_command(bgp)
     from .bgp_quagga_v6 import bgp
     ipv6.add_command(bgp)
 elif routing_stack == "frr":
-    from .bgp_frr_v4 import bgp
-    cli.add_command(bgp)
-    from .bgp_frr_v6 import bgp
-    cli.add_command(bgp)
+    @cli.command()
+    @click.argument('bgp_args', nargs = -1, required = False)
+    def bgp(bgp_args):
+        bgp_cmd = "show bgp"
```

Comment:
Please add a command description for both "show bgp" and "clear bgp". 

We had this before for clear: """Clear BGP peers / state""" and something similar for show.